dot/services.go

```diff
@@ -169,6 +168,8 @@ func createRuntime(cfg *Config, ns runtime.NodeStorage, st *state.Service,
 		if err != nil {
 			return nil, fmt.Errorf("failed to create runtime executor: %s", err)
 		}
+	default:
+		return nil, fmt.Errorf("unknown wasm interpreter: %s, failed to create runtime", cfg.Core.WasmInterpreter)
```

Comment:
use a sentinel error and wrap it i.e.

```go
var ErrWasmInterpreterName = errors.New("unknown was interpreter name")
```

```suggestion
		return nil, fmt.Errorf("%w: %s", ErrWasmInterpreterName, cfg.Core.WasmInterpreter)
```

and no need to mention `failed to create runtime`, this context should wrap the error returned by `createRuntime` instead.

Reply:
Updated.